src/components/rockets/Rocket.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import Card from '../../Components/rockets/Card';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Please kindly delete this comment line to have a clean and professional codebase, and if you have linters error it better to fix them instead of disable them 😁 🙃